mappings/net/minecraft/entity/player/PlayerEntity.mapping

```diff
@@ -154,8 +182,11 @@ CLASS net/minecraft/class_1657 net/minecraft/entity/player/PlayerEntity
 	METHOD method_7327 getScoreboard ()Lnet/minecraft/class_269;
 	METHOD method_7328 dropItem (Lnet/minecraft/class_1799;Z)Lnet/minecraft/class_1542;
 		ARG 1 stack
+		ARG 2 retainOwnership
 	METHOD method_7329 dropItem (Lnet/minecraft/class_1799;ZZ)Lnet/minecraft/class_1542;
 		ARG 1 stack
+		ARG 2 spreadItems
```

Comment:
Although this is technically used to spread items around when dying, this could be used on a single stack - in which case it sounds like it would split the stack. I think `randomToss` would be a little more explicit maybe ?

Reply:
The method only ever takes one stack. `randomToss` works, I supposed, but I can't help but wonder if there's a better term to describe it. `randomToss` sounds too Engrish.

Reply:
`spread` should suffice (with comment)

Reply:
Not sure what "too engrish" means here, but if you want conciseness "spill" works fine.

Reply:
I think `spreadItems` is the clearest out of all of these. I immediately understand what it means, but I wouldn't know what "spill" or "randomToss" mean.

Reply:
Let's continue using spreadItems and add a jd comment on what it means.

Reply:
I very much disagree. When I saw `spreadItems`, my first thought was "wait, there is a way to automatically split a stack when dropping it ?".

Reply:
Wait a minute. So this doesn't split the stack to throw the individual items in random directions? In that case, I agree `spreadItems` is a very bad name for this.

Reply:
Nop, it's for adding random velocity when dropping the item (eg. For spreading the stacks around you when you die)

Reply:
`spew`

Reply:
`throwRandomly`

Reply:
I think "scatter" is already used in yarn (`ItemScatterer` etc).

Reply:
But it doesn't scatter the items in the stack! It only throws the stack, as a single entity, in a random direction.

Reply:
That's what `ItemScatterer` does as well.

Reply:
No, `ItemScatterer` takes a list of stacks and scatters them in random directions.

Reply:
I'm partial towards `spill`

Reply:
`ItemScatterer` -> `ItemSpiller` 🤔